scripts/update-git-platinum.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+## This is an update script for /data/git-platinum. It's meant to be executed
+## from the root of the repo.
+
+TARBALL_PATH=data/git-platinum.tgz
+
+# This is here in case the last script run failed and it never cleaned up.
+# Is there a better way to handle this?
+rm -rf git-platinum
```

Comment:
Maybe a safer thing to do here is checking if the directory exists and asking the person to remove it and exiting the script.

I'm worried that we'd remove a `git-platinum` by accident, e.g. we ran the script in the wrong directory :sweat_smile:

Reply:
What if the whole script worked in some sort of `.workdir` and we added that one to `.gitignore`?

Reply:
https://github.com/radicle-dev/radicle-surf/pull/180#discussion_r618214499 would be enough to resolve this I think :)

Reply:
I still created a sort of workdir. Feels cleaner anyway. How's that look?